open_academy/wizards/wizard_save_attendee.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models, fields, Command
+
+
+class Wizard(models.TransientModel):
+    _name = 'wizard_save_attendee'
+    _description = 'Save attendees'
+    session_ids = fields.Many2many(
+        'session',
+        required=True,
+        default=lambda self: self.env['session'].browse(self._context.get('active_id'))
```

Comment:
Remove this, please send the ids on the action call the ids via context with `default_session_ids` key